Gems/ScriptCanvas/Code/Include/ScriptCanvas/Grammar/AbstractCodeModel.cpp

```diff
@@ -2650,6 +2652,19 @@ namespace ScriptCanvas
             execution->SetSymbol(Symbol::Break);
         }
 
+        void AbstractCodeModel::ParseComponentExtension()
```

Comment:
Here, the AbstractCodeModel detects whether the graph can only be used with the Entity/Component system.